src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,29 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, setSelectedTab }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            key={tab.id}
+            data-cy="Tab"
+            className={cn({ 'is-active': selectedTab === tab })}
```

Comment:
```suggestion
            className={cn({ 'is-active': selectedTab === tab.id })}
```

Reply:
I`m not sure what you meant, but I changed logic to comparing by id.